agent/config/parse_windows.go

```diff
@@ -32,6 +34,11 @@ const (
 	// to skip the windows server version check. This is useful for testing and
 	// should not be set for any non-test use-case.
 	envSkipWindowsServerVersionCheck = "ZZZ_SKIP_WINDOWS_SERVER_VERSION_CHECK_NOT_SUPPORTED_IN_PRODUCTION"
+	gmsaPluginGUID                   = "{859E1386-BDB4-49E8-85C7-3070B13920E1}"
```

Comment:
Do we ever expect this GUID to change (for example after we upgrade the plugin version)

Reply:
Even if the plugin version upgrades, we do not expect this GUID to change. For example, we have already changed the plugin twice and kept the GUID constant. The input to the plugin is a JSON field that is deserialized, so the plugin is also extensible and backwards compatible. 


We have published this GUID to customers as well (for the EKS domainless gMSA already released), so changing its GUID would be breaking behavior.